Brocfile.js

```diff
@@ -40,6 +41,12 @@ var tree = replace( app.toTree(), {
 // please specify an object with the list of modules as keys
 // along with the exports of each module as its value.
 
+if ( !isProduction ) {
```

Comment:
The check is not needed - refer to example in https://github.com/softlayer/sl-ember-components/blob/develop/Brocfile.js